script/Arcueid/Day 5/05_00_ARC05_5.txt

```diff
@@ -499,32 +499,34 @@ And so, with all the determination I can muster, I glare at her, my body shudder
 [sha:1a56a7d27005550a64dd595917f088c3ba533963]{
 -- Page 409, Offset 9664.
 -- 「……はあ。うん、方針に変更なしなのね。
-"...%{i}Sigh%{/i}, it seems that I can't make you change your mind.
+"...%{i}Sigh%{/i}, seems like there's no changing your mind, huh.
 }
 [sha:8811325fc176c02195fda31f0221a6d39edcaa49]{
 -- Page 409, Offset 9665.
 -- これを見ても気が変わらないなんて、あなたもヴローヴのことは言えないかも。
-To think that you'd still fight after seeing that. Perhaps I was wrong about both you and Vlov.
+// check on second sentence
```

Comment:
not too sure myself, but
I'm interpreting lit. "about ((you) and (vlov's nature)), cannot say i guess." which reads to me like the old version.